### PR TITLE
small fix to hover.py example

### PR DESCRIPTION
**Pull request by tp199911**
fix where %x formatting gives TypeError due to np.floor being of type float

### CODE REVIEW

**Comment by bryevdv** [comment]
Oh right, python 3.5 broke this. I think this would work (have not had a change to test) but if so it would be preferable, I think:
```
colors = [
    "#%02x%02x%02x" % (int(r), int(g), 150) for r, g in zip(np.floor(50+2*x), np.floor(30+2*y))
]
```
There are probably other examples that need a similar update, I think `color_scatter.py`, plus notebook/server versions of the same. @tp199911 would you be amendable to rounding out the work on this PR?

**Comment by tp199911** [comment]
@bryevdv, it is done. I am leaning towards the .astype solution myself, since it leaves everything nicely vectorized, but in this case your latter solution is just more readable, so I went with that.

**Comment by birdsarah** [comment]
@tp199911 we just had a big change on master so you'll need to pull that into this pr.

**Comment by tp199911** [comment]
@birdsarah Done!

**Comment by tp199911** [comment]
I don't understand why the tests failed on this new commit; all it did was rm the 3 examples that are also gone in master. Any ideas?

**Comment by birdsarah** [comment]
@tp199911 - did you look at the merge conflict and then remove the files or did you merge in the latest master into your branch? 

I'm asking because I'm not seeing the latest commits from master on your branch.

You need to bring your branch up to date with master (which will tell travis to install futures which is why your tests are failing). https://help.github.com/articles/syncing-a-fork/

**Comment by tp199911** [comment]
@birdsarah I'm a bit at a loss now...

**Comment by bryevdv** [comment]
Maybe a problem with a stale TravisCI cache? I cleared the cache for this PR and restarted. I think there may  actually be a problem on master at the moment too. We will get it sorted out soon, thanks for your patience @tp199911 !

**Comment by birdsarah** [comment]
awesome - thanks for bearing with us @tp199911!

There's a small problem with the PR. You manually changed the ipython notebook and it's no longer valid json. (see my note inline)

I'm just testing the doc updates now.

File: examples/plotting/notebook/color_scatterplot.ipynb [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "x = np.random.random(size=N) * 100\n",
     "y = np.random.random(size=N) * 100\n",
     "radii = np.random.random(size=N) * 1.5\n",
-    "colors = [\"#%02x%02x%02x\" % (r, g, 150) for r, g in zip(np.floor(50+2*x), np.floor(30+2*y))]"
+    "colors = ["#%02x%02x%02x" % (int(r), int(g), 150) for r, g in zip(50+2*x, 30+2*y)]"
```
Comment (birdsarah) [inline examples/plotting/notebook/color_scatterplot.ipynb]: this change results in the notebook not being able to open with the following error: `Unreadable Notebook: /Users/caged/Dev/bokeh/bokeh/examples/plotting/notebook/color_scatterplot.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n  {\\n   "cell_type": "r...',)`

I'm guessing you need to escape the `"`

**Comment by birdsarah** [comment]
I'm having a hard time building the docs today, but the code looks fine and I've tested the code separately and it works great - I'm sure they'll be fine.

I've also tested the examples on python27.

So looking good to me apart from the notebook fix.

**Comment by tp199911** [comment]
@birdsarah Indeed, the quotes inside needed to be escaped. Still no-go from Travis, but I guess that's because of the same reason in master?

**Comment by bryevdv** [comment]
@tp199911 @birdsarah this is marked WIP, is it ready?

**Comment by damianavila** [comment]
I fixed this one in my py35 branch... but this one seems more simple fix... LGTM

**Comment by bryevdv** [comment]
OK I will assume it is ready and merge.

**Comment by bryevdv** [comment]
Thanks @tp199911 !

**Comment by birdsarah** [comment]
Thanks @tp199911 - sorry I missed the second round of testing this.

**Pull request merged**